packages/webapp/pages/signup.tsx

```diff
@@ -34,7 +32,6 @@ function Signup(): ReactElement {
             trigger="login page"
             onClose={onClose}
             onSuccessfulLogin={onClose}
-            version={authVersion}
```

Comment:
Just make sure this isn't passed for tracking somewhere?

Reply:
Yep. This is for layout purposes which we cleaned up. Also, I have removed the prop in the interface which typescript would show if there is any error.